### DISCUSSION TITLE
Verbatim Module Syntax implemented by default

### DISCUSSION THREAD

**Discussion by louiss0** [withastro/roadmap#634] [Proposal]
### Body

# Summary

I believe Astro need to ship  in a `tsconfig.json` file `"verbatimModuleSyntax": true` in all templates created by `Astro`.
This setting is a necessary rule 

# Background & Motivation

If you don't know when you set `"verbatimModuleSyntax": true`  tells typescript to throw an error when a value is used as a type  
but not implemented as a type. I was in a project where I had to write this setting myself. When I created a new project with astro I had to implement this config myself. If I remember correctly all tsconfig's ship with `"importsNotUsedAsValues": true`
 
# Goals

- Make sure that all templates have the `"verbatimModuleSyntax": true`  so that ts code is written properly.

**Comment by bluwy** [comment]
It should already be set to that in 3.0: https://github.com/withastro/astro/blob/next/packages/astro/tsconfigs/base.json
